City-of-Helsinki/drupal-helfi-etusivu
# UHF-5618 react news rss

# [UHF-5618](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5618)

## What was done
Fallback rss feed for react

## How to install
* Use database from testing environment unless you want to create bunch of news items manually
* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-5618_react_news_rss`
  * `make fresh`
* Run `make drush-cr`
* Make sure you have elasticsearch running & indexed

## How to test
* Check elasticsearch index `localhost:9200/_search?size=100&pretty=true`
* Go to /fi/news/react/rss 
  * You should see all news filtered by language
* Test changing language
  * You should see news filtered by language
* Test query parameters (examples: neighbourhoods: kruununhaka, kluuvi. topics: kaupunki ja hallinto. groups: nuoret, yritykset )
  * Neighbourhoods, topics, groups
  * You should see news filtered by query parameters
